### DISCUSSION TITLE
Scrolling for fzf-lua git* commands

### DISCUSSION THREAD

**Discussion by skoch13** [LazyVim/LazyVim#3895] [Q&A] [answered]
Using fzf-lua `commits` command has different set of keybinding for scrolling: `C-f` / `C-b` scroll the list, not the preview. Is it intentional?

**Comment by dpetka2001** [comment]
It seems that preview-down/up doesn't work in `git_commits`. Just tried it in a non-Lazyvim setup and `<Shift-down/up>` (defaults for preview down/up) don't work either. Better report it in fzf-lua repo if you'd like.

**Comment by dpetka2001** [comment, marked as answer]
@skoch13 Additional configuration was needed after all. Please see #3904
